app/src/main/java/org/dhis2/usescases/settings/SettingsRepository.kt

```diff
@@ -110,8 +110,8 @@ class SettingsRepository(
                 smsConfig.gateway,
                 smsConfig.resultSender,
                 smsConfig.resultWaitingTimeout,
-                generalSettings?.numberSmsToSend() == null,
```

Comment:
Commented code should be keep for any reason?

Reply:
I forgot to remove the comment. I did it to be able to edit the sms configuration.